src/Storage/Storage.Management/help/New-AzStorageContext.md

```diff
@@ -163,14 +187,59 @@ PS C:\> $Context = New-AzStorageContext -StorageAccountName "myaccountname" -Use
 
 This command creates a context by using the OAuth (Azure AD) Authentication.
 
+### Example 11: Create a context by specifying a storage account name, storage account key and custom blob endpoint
+```
+PS C:\> New-AzStorageContext -StorageAccountName "MyAccountName" -StorageAccountKey "< Storage Key for MyAccountName ends with == >" -BlobEndpoint "https://MyAccountName.blob.core.windows.net/"
+```
+
+This command creates a context for the account named MyAccountName with a key for the account, and specified blob endpoint and table endpoint.
+
+### Example 12: Create a context for an anonymous storage accouont with specified file and queue endpoints 
+```
+PS C:\> New-AzStorageContext -StorageAccountName "MyAccountName" -Anonymous -Protocol "http" -FileEndpoint "https://MyAccountName.file.core.windows.net/" -QueueEndpoint "https://MyAccountName.queue.core.windows.net/"
+```
+
+This command creates a context for anonymous use for the account named ContosoGeneral, with specified file and queue endpoints.
+
+### Example 13: Create a context by using an SAS token with specified endpoints
+```
+PS C:\>$SasToken = New-AzStorageContainerSASToken -Name "MyContainer" -Permission "rad"
+PS C:\> New-AzStorageContext -StorageAccountName "MyAccountName" -SasToken $SasToken -BlobEndpoint "https://MyAccountName.blob.core.windows.net/" -TableEndpoint "https://MyAccountName.table.core.windows.net/" -FileEndpoint "https://MyAccountName.file.core.windows.net/" -QueueEndpoint "https://MyAccountName.queue.core.windows.net/"
+```
+
+The first command generates an SAS token by using the New-AzStorageContainerSASToken cmdlet for the container named MyContainer, and then stores that token in the $SasToken variable.
+The second command creates a context for the account named ContosoGeneral that uses the SAS token and a specified blob endpoint, table endpoint, file endpoint, and queue endpoint. 
+
+### Example 14: Create ea context by using the OAuth Authentication with a specified blob endpoint 
+```
+PS C:\> New-AzStorageContext -UseConnectedAccount -BlobEndpoint  "https://MyAccountName.blob.core.windows.net/"
+```
+
+This command creates a context by using the OAuth authentication with a specified blob endpoint.
+
 ## PARAMETERS
 
 ### -Anonymous
 Indicates that this cmdlet creates an Azure Storage context for anonymous logon.
 
 ```yaml
 Type: System.Management.Automation.SwitchParameter
-Parameter Sets: AnonymousAccount, AnonymousAccountEnvironment
+Parameter Sets: AnonymousAccount, AnonymousAccountEnvironment, AnonymousAccountSeviceEndpoint
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -BlobEndpoint
+Azure storage endpoint
```

Comment:
Please use: "Azure storage blob service endpoint."
Same as below for F/Q/T

Reply:
Done